org-eclipse-che-jdt-ls-extension/jdt.ls.extension.api/src/main/java/org/eclipse/che/jdt/ls/extension/api/dto/OrganizeImportsResult.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.jdt.ls.extension.api.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.eclipse.lsp4j.WorkspaceEdit;
+
+/** @author Anatolii Bazko */
+public class OrganizeImportsResult {
+  private List<ImportConflicts> importConflicts;
```

Comment:
I think a List<List<String>> would be sufficient. They are not really conflicts, either, I "ambigousTypes" may be a more appropriate word.